### PR TITLE
Add:fly pg attachments command

### PR DESCRIPTION
**Pull request by rugwirobaker**
* Consolidates the likes of attach and detach.
* Adds a new list attachments command.
* There old commands `fly pg attach` and `fly pg detach` are still there for now.

### CODE REVIEW

File: internal/command/postgres/attachments.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+		short = "manage database attachments to apps"
+		long  = short + "\n"
+		usage = "attachments [command]"
+	)
+
+	cmd := command.New(usage, short, long, nil)
+
+	cmd.AddCommand(
+		newAttachmentsList(),
+		newAttach(),
```
Comment (davissp14) [inline internal/command/postgres/attachments.go:28]: Do you think we should rename the attach/detach sub commands with this? 

```
fly pg attachments attach` doesn't feel quite right.

**Review by davissp14 — CHANGES_REQUESTED** [review]
I added a quick note, but we may want to consider renaming the subcommands. 

This change will also require updates to our documentation.

File: internal/command/postgres/attachments.go (hunk 1 above)
Comment (rugwirobaker) [reply internal/command/postgres/attachments.go:28]: @davissp14 I think people are too used to it and it makes sense. I'd rather find a new name for the new attachments command.

**Comment by guillaumervls** [comment]
@rugwirobaker what's in the way of us merging / shipping / closing this? How can I help?

**Comment by dangra** [comment]
Closing, we can reopen if we ever revisit the idea.

**Pull request closed without merging**